File: Day1/Puzzel1.py
with open("input.txt") as input_file:
    input = input_file.read().splitlines()
input.append("")

max = -1;
count = 0

for line in input:
    if line == "":
        if count > max:
            max = count
        count = 0
    else:
        count += int(line)

print(max)

File: 2022/Day01/Puzzel1.cpp
#include <iostream>
#include <fstream>
#include <string>

int main(){
    std::ifstream inputFile("input.txt");
    std::string temp;
    int max=0;
    int count = 0;
    while( std::getline(inputFile, temp) )
    {
        if ( temp == "" )            
        {
            if ( count > max )
                max = count;  
                count = 0;
        }
        else
          count += std::stoi(temp);
    }
    std::cout << "The maximum value = " <<  max << '\n';
}
